pkg/providers/vault/interfaces.go

```diff
@@ -20,3 +20,12 @@ import "net/http"
 type Caller interface {
 	Do(req *http.Request) (*http.Response, error)
 }
+
+type loggingClient interface {
```

Comment:
Could loggingClient be exported?

Reply:
does not have to be

Reply:
It's a private package-level concern and shouldn't be exported.